src/main/kotlin/org/jetbrains/research/testspark/data/ProjectContext.kt

```diff
@@ -0,0 +1,19 @@
+package org.jetbrains.research.testspark.data
+
+import com.intellij.openapi.module.Module
+import com.intellij.psi.PsiClass
+
+data class ProjectContext(
+    // The class path of the project.
+    var projectClassPath: String? = null,
+
+    // The URL of the file being tested.
+    var fileUrl: String? = null,
```

Comment:
Do I understand correctly that this is `File(path).toURI().toURL().toString()`? It is named URL but its type is `String`.